src/main/java/frc/robot/RobotContainer.java

```diff
@@ -130,4 +136,12 @@ public Command getAutonomousCommand() {
     // An example command will be run in autonomous
     return null;
   }
+
+  public Command getChargeStationRoutine(){
+    return Commands.sequence(m_pickupItemCommand,  new TurnPID(m_driveTrain, -(m_photonVision.getTarget().getYaw())), m_driveMetersCommand,  m_dropItemCommand, m_driveMetersCommand);
+  }
+
+  public Command getPickupRoutine(){
+    return Commands.sequence(m_pickupItemCommand, new TurnPID(m_driveTrain, -(m_photonVision.getTarget().getYaw())), m_driveMetersCommand, m_dropItemCommand, new TurnPID(m_driveTrain, -(m_photonVision.getTarget().getYaw())), m_driveMetersCommand, new TurnPID(m_driveTrain, 180), m_driveMetersCommand);
```

Comment:
Need to add values to drive drive meters command, not just call the command. Maybe create a new command with `new DriveMetersCommand()` like the turnPID is doing it